### ISSUE TITLE
Необрабатываемое исключение: [Введите заголовок]

### ISSUE THREAD

**Issue by severrn** [remyroy/CDDA-Game-Launcher#246]
* Описание: [Напишите, что вы делали и что произошло]
* Версия: 1.3.16
* OS: Windows-7-6.1.7601-SP1 (64-bit)
* Тип: `<class 'pywintypes.com_error'>`
* Значение: (-2144927744, 'OLE error 0x80270000', None, None)
* Диагностика:
```
  File "cddagl\ui.py", line 6953, in download_http_finished
  File "cddagl\ui.py", line 173, in delete_path
  File "site-packages\winutils.py", line 77, in delete
  File "site-packages\winutils.py", line 40, in _file_operation

```

**Comment by severrn** [comment]
Win 7, yes. I can translate it to english if you need, but there is nothing important anyway

**Comment by severrn** [comment]
> Do you have the English version?

CDDA Game Launcher version: 1.3.16
OS: Windows-7-6.1.7601-SP1 (64-bit)
Type: <class 'pywintypes.com_error'>
Value: (-2144927744, 'OLE error 0x80270000', None, None)
Traceback:
File "cddagl\ui.py", line 6953, in download_http_finished

File "cddagl\ui.py", line 173, in delete_path
File "site-packages\winutils.py", line 77, in delete
File "site-packages\winutils.py", line 40, in _file_operation

**Comment by remyroy** [comment]
This should be fixed with fdf9488b9f18fe7322a66b22dfefc48dee048648 . It will be part of the next release.
